stronk/models/program.py

```diff
@@ -47,6 +52,10 @@ def create(author, name, duration, description):
     def find_by_id(id):
         return Program.query.filter_by(id=id).first()
 
+    @staticmethod
+    def find_by_author(author):
+        return Program.query.filter_by(author=author)
```

Comment:
oh wow I'm surprised you don't need either a `all()` or `first()` here

Reply:
wait I forgot to add the `all()` haha